### DISCUSSION TITLE
NGINX not starting

### DISCUSSION THREAD

**Discussion by AlanBarnes** [beyondcode/herd-community#858] [Q&A]
When I start HERD everything starts but NGINX. I get the following error in the log -

2024/07/04 11:20:47 [emerg] 1292#17208: bind() to 127.0.0.1:80 failed (10013: An attempt was made to access a socket in a way forbidden by its access permissions)

I am new to herd and coding, so I am trying to figure it out. It was working fine 2 days ago when I was working through a laracast tutorial. I have already tried to uninstall and reinstall to no avail. I have a pro license.

**Comment by LTIOfficial** [comment]
I find it rather unsettling that no one has responded to Mr. Barnes over all of this time.  The open-source community, as a whole, used to trip all over each other trying to share experience with others to help them through seemingly (to them) difficult challenges.  You might say it's how the community "nurtured its young" in order to grow into where it is today.  If the 'norm' is to now ignore those facing challenges, ask yourself what that says about us.  \<sigh /\>



For Mr. Barnes (if you haven't quit trying to make it work by now),
It could be that you are already running another http server on port 80, especially if you are coming to Herd from something like WAMP, MAMP, or XAMPP.  All three of those typically fire up "apache" on port 80/443 whenever you boot or reboot.

So if you're running one of those (or similar), open its control screen and turn off any instances of apache or nginx.  You'll also want to prevent it from starting that up on boot/reboot to keep it out (temporarily?) of the way while you're taking Herd for a test drive.  On the other hand, Herd should be able to take advantage of WAMP/MAMP/XAMPP's mysql/mariadb database server, so don't stop that if you sometimes need more than sqlite.
